### ISSUE TITLE
0.0.0.0

### ISSUE THREAD

**Issue by God-damnit-all** [richardszalay/pshosts#19]
Elaborated on here: https://superuser.com/questions/1030329/better-to-block-a-host-to-0-0-0-0-than-to-127-0-0-1

The short version: Windows designates 0.0.0.0 as a non-routable meta-address used to designate an invalid, unknown, or non-applicable target. It is not subject to timeout resolution and does not interfere with local web servers.

**Comment by richardszalay** [comment]
The intention of `-Loopback` isn't to block the host, it's to route it to the local machine via the loopback address.

I wasn't aware of using `0.0.0.0` to block hosts, but it shouldn't be napped to the `-Loopback` switch. Perhaps a `-Block` instead (ideally it would be named after the correct term for that address, if you can find that out).

**Comment by richardszalay** [comment]
After some quick reading it looks like `-NonRoutable` seems like the best option ("Block" has multiple interpretations).

**Comment by God-damnit-all** [comment]
Good point. I guess I'm just used to using the hosts file only for blocking, so I wasn't really thinking about it in that context.

I think 0.0.0.0 actually is a loopback address in Linux, but when used in this manner in the hosts file, becomes a non-routable address.
